File: src/main/java/no/fintlabs/access/control/repository/FieldAccessEntityRepository.java
package no.fintlabs.access.control.repository;

import no.fintlabs.access.control.model.entity.FieldAccessEntity;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.Collection;

public interface FieldAccessEntityRepository extends JpaRepository<FieldAccessEntity, Integer> {
    Collection<FieldAccessEntity> findByResourceId(Integer id);
}

File: src/main/java/no/fintlabs/access/control/repository/FieldAccessEntityRepository.kt
package no.fintlabs.access.control.repository

import no.fintlabs.access.control.model.entity.FieldAccessEntity
import org.springframework.data.jpa.repository.JpaRepository

interface FieldAccessEntityRepository : JpaRepository<FieldAccessEntity, Int> {
    fun findByResourceId(id: Int): Collection<FieldAccessEntity>
}
